app/controllers/User.scala

```diff
@@ -124,6 +124,7 @@ final class User(
                 _ <- env.tournament.cached.nameCache preloadMany {
                   pag.currentPageResults.flatMap(_.tournamentId).map(_ -> ctxLang)
                 }
+                notes <- ctx.me.nonEmpty ?? { env.round.noteApi.byGameIds(pag.currentPageResults.map(_.id), ctx.me.get.id) }
```

Comment:
please don't do `me.nonEmpty ?? ... me.get`, it's an anti-pattern.

Use `ctx.me ?? { me => ...}` instead

Reply:
got it, fixed with latest commit